seonshine_backend/middleware/validation/userValidate.js

```diff
@@ -99,3 +99,32 @@ export const validateGetUserDetail = (req, res, next) => {
   }
   next();
 };
+
+export const validateChangeAvatar = async (req, res, next) => {
+  const allowedFileTypes = ["image/jpeg", "image/jpg", "image/png"];
+  const maxSize = 5 * 1024 * 1024;
+  try {
+    const file = req.file;
+
```

Comment:
please check:
1. Admin can change avatar for all of user
2. Only normal user/ restaurant can change his avatar